modules/ROOT/pages/secret-group-add-tls-context-flex.adoc

```diff
@@ -0,0 +1,16 @@
+= Add a TLS Context to Flex Gateway Connected Mode
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+NOTE: The following information as well as how to apply a TLS context to an API Instance is also included in the Flex Gateway documentation. For more information, see xref:gateway::flex-conn-tls-config.adoc[Configuring TLS Context for Flex Gateway Connected Mode].
```

Comment:
Is this note really needed? It seems it is just saying that this info also exists in other article, and the link shared is also part of the See Also section, so it just seems to be duplicating content. Please check if it can be removed.